coeus-s2sgen-impl/src/main/java/org/kuali/coeus/s2sgen/impl/validate/S2SErrorHandlerService.java

```diff
@@ -5,4 +5,6 @@
 public interface S2SErrorHandlerService {
 
     AuditError getError(String key);
+    
+    AuditError getErrors(String key, String formName);
```

Comment:
Please convert all generators to use your new api.  The form name should be available in each usage case.